src/api/test/functional/backend_test.rb

```diff
@@ -21,7 +21,7 @@ def test_validate_bsxml
           'productlist', 'binary_released', 'check', 'required_checks', 'status_report',
           'staged_requests', 'remove_staged_requests', 'status_ok', 'staging_project',
           'staging_projects', 'create_staging_workflow', 'update_staging_workflow', 'accept_staging_projects',
-          'excluded_requests', 'create_excluded_requests', 'delete_excluded_requests', 'create_staging_projects', 'backlog'].include?(schema)
+          'excluded_requests', 'create_excluded_requests', 'delete_excluded_requests', 'create_staging_projects', 'backlog', 'tokenlist'].include?(schema)
```

Comment:
@mlschroe I am not sure if there is a backend schema for the tokenlist. Do you happen to know that?